.github/workflows/build-production.yml

```diff
@@ -0,0 +1,135 @@
+name: Build Production
+
+on:
+  push:
+    tags:
+      - v*.*.*
+    paths-ignore:
```

Comment:
The paths-ignore directive is not required for this since the workflow only runs on releases which are intentionally triggered, presumably on an actual change to the code